apps/fabric-website/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

```diff
@@ -41,9 +43,13 @@ const MyIconButton = () => <IconButton iconProps={{ iconName: 'Add' }} title="Ad
 ReactDOM.render(<MyIconButton />, document.body.firstChild);
 ```
 
-### Fluent UI React: SVG-based icons
+### Fluent UI React: SVG-based icons (open source as of version 8)
+
+The Fluent UI SVG-based icon set is open source and released under the MIT license.
 
-Fluent UI recently released an SVG-based icon set which allows you to import and bundle only the icons you need. This results in smaller bundle sizes compared to the font-based approach with `initializeIcons`, which downloads all icons by default.
+Fluent UI released an SVG-based icon set which allows you to import and bundle only the icons you need. This results in smaller bundle sizes compared to the font-based approach with `initializeIcons`, which downloads all icons by default.
+
+In `@fluentui/react` version 8, we are enabling more widespread use by open sourcing the core icon set. Be aware that any branded icons have been removed from `@fluentui/react-icons-mdl2`. If you need branded icons, please refer to the package `@fluentui/react-icons-mdl2-branded`.
```

Comment:
If we're delayed in releasing the branded package we should temporarily remove the mention.
```suggestion
In `@fluentui/react` version 8, we are enabling more widespread use of icons by open-sourcing the core icon set. Be aware that any branded icons have been removed from `@fluentui/react-icons-mdl2`.
```